canonical-websites/build.snapcraft.io
# Remove pagination in 'Add repos' dialog

## Done

Added loading all pages of repositories automatically and showing them all as one scrollable list in 'Add repos' dialog.

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Sign in and go to 'Add repos'
- All your repositories should be loaded into one long list without pagination


## Issue / Card

Fixes #383
